app/layout.tsx

```diff
@@ -20,6 +21,7 @@ const RootLayout = ({
       <main>
         <Suspense fallback={<Loading />}>{children}</Suspense>
       </main>
+      <Newsletter />
```

Comment:
The Newsletter shouldn't be part of the layout. It is not a component that should appear on all pages.
Here we have the Navigation and the Footer, because they will appear on all pages.

Reply:
Okay i change,  i put in page.tsx file under BlogList